### DISCUSSION TITLE
There is always a blue dot on notifications button

### DISCUSSION THREAD

**Discussion by shalva97** [community/community#113262] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Some time ago there was a Chinese data leak here https://github.com/I-S00N/I-S00N and I added few comments.

The problem is that it got blocked and I have some unread notifications, but it does not show up in notifications and shows a blue dot permanently.
![Screenshot 2024-03-19 at 13 11 23](https://github.com/community/community/assets/22417494/7918167e-1d97-48f9-a966-aafce6694f64)

![Screenshot 2024-03-19 at 13 09 43](https://github.com/community/community/assets/22417494/c13381fe-c8cb-492c-891c-469c102d05e7)
![Screenshot 2024-03-19 at 13 11 43](https://github.com/community/community/assets/22417494/ad05cb35-83de-438b-8fda-2802af5e7264)

Any ideas how to mark it as read? so that this blue dot will show up only when I actually have a new notification

**Comment by AidenJamesYt** [comment]
your just in the repository notifications

check "inbox"

**Reply by shalva97** [reply]
Can you clarify what do you mean by check inbox? I have inbox selected in the second pic, but nothing is showing up.

**Reply by KnightChaser** [reply]
Have the same issue, but I don't get what do you mean.

**Comment by KnightChaser** [comment, marked as answer]
That is one of the common ghost notification methods, able to clean such notifications with the `curl` command via GitHub API. Refer to the following answer, it worked for my case(I also had trouble with that `I-S00N` repository)

https://github.com/orgs/community/discussions/6874#discussioncomment-2859125

**Comment by ebndev** [comment]
I found that this might be the issue: if you are marking notifications "Done" before "Mark as Read".

See this [Discussion](https://github.com/orgs/community/discussions/32380#discussioncomment-3785542) / comment for more.

**Comment by linker34** [comment]
right - you can fix it up through our given information on our tab

**Comment by linker34** [comment]
Can you clarify what do you mean by check inbox? I have inbox selected in the second pic, but nothing is showing up on [webiste](https://thepoolapk.com/)

**Reply by KnightChaser** [reply]
![image](https://github.com/community/community/assets/47494005/3ff22d60-16c7-4ad5-903c-cd4ac232d07b)
Are you sure with that?

**Reply by shalva97** [reply]
@KnightChaser nice waifu m8

**Reply by KnightChaser** [reply]
> @KnightChaser nice waifu m8

Yeah she's nice to me

**Comment by PavanLuca** [comment]
![immagine](https://github.com/community/community/assets/123581658/20bee99d-4217-4f2b-b8e1-b05c1757e09e)

**Comment by sgf** [comment]
i have the same problem.
